.bazelrc

```diff
@@ -33,6 +33,10 @@ build --features swift.index_while_building
 # Prevents leaking unexpected binaries via PATH to tests
 build --test_env=PATH=/usr/bin:/bin:/usr/sbin:/sbin
 
+# Avoid running xcode-locator which can be super slow if the machine has many
+# Xcode versions
+build --xcode_version_config=//:host_xcodes
```

Comment:
So IIUC this prevents the 
```
Error in fail: Generating xcode-locator-bin failed.  return code 256, stderr: Timed out, stdout: 
```
coming from the toolchain that includes `tools/toolchains/xcode_configure/xcode_configure.bzl`. @jerrymarino I see you added that correct? Any implications here?

It would be nice to prevent this annoying CI flakes indeed.